packages/client/hmi-client/src/components/data-explorer/search-by-example.vue

```diff
@@ -0,0 +1,203 @@
+import { ResultType, SearchByExampleOptions } from '@/types/common';
+import { Model } from '@/types/Model';
+import { XDDArticle } from '@/types/XDD';
+import Button from '@/components/Button.vue';
+
+const props = defineProps({
+	item: {
+		type: Object as PropType<ResultType | null>,
+		default: () => null
+	}
+});
+
+const emit = defineEmits(['hide', 'search']);
+
+const searchOptions = ref<SearchByExampleOptions>({
+	similarContent: true,
+	forwardCitation: false,
+	bakcwardCitation: false,
+	relatedContent: false
+});
+
+const onClose = () => {
+	emit('hide');
+};
+
+function search() {
+	emit('search', searchOptions.value);
+	onClose();
+}
+
+const isSearchDisabled = computed(() => {
+	if (!props.item) return true;
+	if (
+		!searchOptions.value.similarContent &&
+		!searchOptions.value.forwardCitation &&
+		!searchOptions.value.bakcwardCitation &&
+		!searchOptions.value.relatedContent
+	) {
+		return true;
+	}
+	return false;
+});
+
+const getTitle = (item: ResultType) => (item as Model).name || (item as XDDArticle).title;
+</script>
+
+<template>
+	<div class="search-by-example-card">
+		<div class="header">
+			<h3>Search by Example</h3>
+			<IconClose16 class="hover-hand" @click="onClose" />
+		</div>
+		<div class="search-items-container">
+			<!--
+				list of dropped items here: this should be rendered component items
+				FIXME: for now, render their title
+			-->
+			<div v-if="props.item">
+				{{ getTitle(props.item) }}
+			</div>
+		</div>
+		<footer>
+			<div @click.stop="searchOptions.similarContent = !searchOptions.similarContent">
+				Similar Content
+				<span v-show="searchOptions.similarContent">
+					<IconCheckboxChecked20 />
+				</span>
+				<span v-show="!searchOptions.similarContent">
+					<IconCheckbox20 />
+				</span>
+			</div>
+			<div @click.stop="searchOptions.forwardCitation = !searchOptions.forwardCitation">
+				Forward Citation
+				<span v-show="searchOptions.forwardCitation">
+					<IconCheckboxChecked20 />
+				</span>
+				<span v-show="!searchOptions.forwardCitation">
+					<IconCheckbox20 />
+				</span>
+			</div>
+			<div @click.stop="searchOptions.bakcwardCitation = !searchOptions.bakcwardCitation">
+				Backward Citation
+				<span v-show="searchOptions.bakcwardCitation">
+					<IconCheckboxChecked20 />
+				</span>
+				<span v-show="!searchOptions.bakcwardCitation">
+					<IconCheckbox20 />
+				</span>
+			</div>
+			<div @click.stop="searchOptions.relatedContent = !searchOptions.relatedContent">
+				Related Content
+				<span v-show="searchOptions.relatedContent">
+					<IconCheckboxChecked20 />
+				</span>
+				<span v-show="!searchOptions.relatedContent">
+					<IconCheckbox20 />
+				</span>
+			</div>
+			<Button :class="{ disabled: isSearchDisabled }" :disabled="isSearchDisabled" @click="search"
+				>Search</Button
+			>
+		</footer>
+	</div>
+</template>
+
+<style scoped>
+.search-by-example-card {
+	border: 1px solid var(--un-color-body-stroke);
```

Comment:
Please replace `--un-*` with the newer CSS properties:
https://github.com/DARPA-ASKEM/TERArium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L834